custom/icds_reports/utils/aggregation_helpers/distributed/base.py

```diff
@@ -72,6 +72,7 @@ class StateBasedAggregationDistributedHelper(BaseICDSAggregationDistributedHelpe
     def aggregate(self, cursor):
```

Comment:
I think you would need to override this method because all the subclasses does not have update_queries function

Reply:
or may be just add the update_queries method in parent class